src/current/v23.2/alter-database.md

```diff
@@ -834,6 +836,120 @@ The `lease_preferences` field is now updated to include `us-west1`.
+- Set the `movr` database to [survive region failures]({% link {{ page.version.version }}/multiregion-survival-goals.md %}#survive-region-failures).
+- Add the remaining regions we defined at cluster start time to the multi-region `movr` database (for a total of 6).
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE movr SET PRIMARY REGION "us-east1";
+ALTER DATABASE movr ADD REGION "us-central1";
+ALTER DATABASE movr ADD REGION "us-west1";
+ALTER DATABASE movr ADD REGION "europe-west1";
+ALTER DATABASE movr ADD REGION "europe-central1";
+ALTER DATABASE movr ADD REGION "europe-east1";
+ALTER DATABASE movr SURVIVE REGION FAILURE;
+~~~
+
+Now that our multiregion cluster is configured, we will use the `ALTER LOCALITY` statements shown below to [configure replication zones]({% link {{ page.version.version }}/configure-replication-zones.md %}) such that:
+
+- [Replicas]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-replica) in the `us-east1` region are only stored on nodes in the US.
+- Replicas in the `europe-central1` region are only stored on nodes in the EU.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE movr ALTER LOCALITY REGIONAL IN "us-east1" CONFIGURE ZONE USING
+    num_replicas = 7,
+    constraints = '{+region=us-east1: 3,
+                    +region=us-central1: 3,
+                    +region=us-west1: 1}',
+    lease_preferences = '[[+region=us-east1], [+region=us-central1]]';
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE movr ALTER LOCALITY REGIONAL IN "europe-central1" CONFIGURE ZONE USING
+    num_replicas = 7,
+    constraints = '{+region=europe-central1: 3,
+                    +region=europe-west1: 3,
+                    +region=europe-east1: 1}',
+    lease_preferences = '[[+region=europe-central1], [+region=europe-west1]]';
+~~~
+
+#### Minimize cross-region write latency
+
+In this example, we use [Zone Config Extensions]({% link {{ page.version.version }}/zone-config-extensions.md %}) to minimize cross-region write latency in a cluster with the following characteristics:
+
+- 3 main US regions (`us-west1`,`us-east1`,`us-central1`).
+- 2 additional distant regions (`europe-west1`, `asia-northeast1`). Only [follower reads]({% link {{ page.version.version }}/follower-reads.md %}) will be made from these regions.
+
+    To minimize the cross-region write latency, we will [configure replication zones]({% link {{ page.version.version }}/configure-replication-zones.md %}) such that the 3 US regions have voting replicas (that is, replicas that participate in [Raft quorum]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft)). This will leave the 2 additional non-US regions with only [non-voting replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#non-voting-replicas).
```

Comment:
```suggestion
To minimize the cross-region write latency, [configure replication zones]({% link {{ page.version.version }}/configure-replication-zones.md %}) such that the 3 US regions have voting replicas (that is, replicas that participate in [Raft quorum]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft)). This will leave the 2 additional non-US regions with only [non-voting replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#non-voting-replicas).
```